### PR TITLE
chore: remove export to client lib menu item for markdown

### LINKED ISSUE #3773: notebooks:  Note panel, tables, and visualizations should not have the option to "Export to Client Library"
While appropriate in Query panel of Notebooks, the "Export to Client Library" is not appropriate for the Note panel and should be removed.

Screen Shot 2022-02-03 at 11 18 51 AM

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #3773

### CODE REVIEW

**Review by drdelambre — CHANGES_REQUESTED** [review]
mixing responsibilities

File: src/flows/components/Sidebar.tsx [hunk 1]
```diff
@@ -239,14 +239,23 @@ const Sidebar: FC = () => {
         {
           title: 'Export to Client Library',
           menu: <ClientList />,
+          disable: () => {
+            if (!flow.data.allIDs.includes(id)) {
+              return true
+            }
+
+            const types = new Set(['markdown'])
```
[the remark this answers is not in the document]
Comment (subirjolly) [reply src/flows/components/Sidebar.tsx:247]: As I mentioned to you over the chat, I started off with checking if `family` match would fit in this situation as that's generic. The issue with that is that `Markdown` belongs to `passThrough` family which some other panels do too. We don't want this menu to be disabled for other panels. As I mentioned, I am open to suggestions.

Is it appropriate to add a family or a family like attribute?

[the remark this answers is not in the document]
Comment (subirjolly) [reply src/flows/components/Sidebar.tsx:247]: possibly another solution can be to register optional `actions`?

**Review by drdelambre — APPROVED** [review]

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3813)

**Pull request merged**